src/pip/_internal/pep425tags.py

```diff
@@ -276,8 +293,16 @@ def get_supported(versions=None, noarch=False, platform=None,
             else:
                 # arch pattern didn't match (?!)
                 arches = [arch]
-        elif platform is None and is_manylinux1_compatible():
-            arches = [arch.replace('linux', 'manylinux1'), arch]
+        elif arch.startswith('manylinux2'):
```

Comment:
FWIW I feel like this is probably a bit of footgun. It feels like the kind of thing that's asking to be a bug once we have a ``manylinux2018`` or something.